### PR TITLE
[Linaro:ARM_CI] Revert adding install of tensorflow-io

### PR DESCRIPTION
**Pull request by elfringham**
Do not install tensorflow-io as an extra package, it is not needed and can cause additional problems and failed to achieve the original intent as it attempted to fix a problem that was actually in the building of the tensorflow-io-gcs-filesystem package for AARCH64

### CODE REVIEW

**Comment by elfringham** [comment]
Probably best to wait for a new release of tensorflow-io before merging this PR

**Comment by elfringham** [comment]
No reason to wait for new release of tensorflow-io. Failures seen in ARM_CI run were not related.

**Review by nitins17 — APPROVED** [review]

**Pull request merged**
